src/getStaticProps.js

```diff
@@ -5,7 +5,7 @@ export async function getStaticProps() {
     props: {
       posts: getAllPostPreviews().map((post) => ({
         title: post.module.meta.title,
-        link: post.link.substr(1),
+        link: post.link,
```

Comment:
Does this change break our comparison logic in `Post.js` here?

https://github.com/tailwindcss/blog/blob/master/src/components/Post.js#L19